app.py

```diff
@@ -227,16 +222,13 @@ def update_graphs(timezone, duration, click_feature, _n_intervals):
                          'baro_pressure': 'Barometric Pressure (mBar)'}, inplace=True)
             sensor_data['Time'] = sensor_data['Time'].dt.tz_convert(timezone)
 
-    return (
-       px.line(sensor_data, x='Time', y='CO₂ (PPM)', color_discrete_sequence=['black'], template='plotly_white',
-               render_mode='svg', hover_data={'CO₂ (PPM)': ':.2f'}),
-       px.line(sensor_data, x='Time', y='Temperature (°C)', color_discrete_sequence=['black'], template='plotly_white',
-               render_mode='svg', hover_data={'Temperature (°C)': ':.2f'}),
-       px.line(sensor_data, x='Time', y='Barometric Pressure (mBar)', color_discrete_sequence=['black'],
-               template='plotly_white', render_mode='svg', hover_data={'Barometric Pressure (mBar)': ':.2f'}),
-       px.line(sensor_data, x='Time', y='Humidity (%)', color_discrete_sequence=['black'], template='plotly_white',
-               render_mode='svg', hover_data={'Humidity (%)': ':.2f'}),
-    )
+    columns_to_plot = ['CO₂ (PPM)', 'Temperature (°C)', 'Barometric Pressure (mBar)', 'Humidity (%)']
+    fig = make_subplots(rows=4, cols=1, shared_xaxes=True)
+    for ind, col in enumerate(columns_to_plot):
+        fig.add_scatter(x=sensor_data["Time"], y=sensor_data[col], mode="lines", line=go.scatter.Line(color="gray"), showlegend=False, row=ind+1, col=1)
```

Comment:
Can you change this to
`fig.add_scatter(x=sensor_data["Time"], y=sensor_data[col], mode="lines", line=go.scatter.Line(color="black"), showlegend=False, row=ind+1, col=1, hovertemplate="Time: %{x}<br>%{text}: %{y:.2f}<extra></extra>", text=[col]*len(sensor_data[col]))`

This adds the hovertemplate to match our existing hover features.